src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudApiService.scala

```diff
@@ -48,9 +48,10 @@ object FireCloudApiService extends LazyLogging {
         if (logger.underlying.isDebugEnabled) {
           logger.debug(e.getMessage, e)
         } else {
-          logger.error(e.getMessage)
+          logger.error(s"$e")
```

Comment:
I guess this is mostly style - I'd prefer `logger.error(e.toString)` over `logger.error(s"$e")`; it feels more straightforward and readable.

Reply:
Updated to `logger.error(e.toString)`